src/main/java/Deadline.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Deadlines are tasks that have a description and must be done by a certain time.
+ */
+public class Deadline extends Task {
+    protected String by;
+
+    /**
+     * Constructor for Deadline task
+     * @param description description of a task that the user inputs
```

Comment:
It would be better to have a full stop at the end of each parameter description.